__tests__/app.test.js

```diff
@@ -73,4 +73,12 @@ describe('getArticleById', () => {
         });
       });
   });
+  test("400: GET /api/articles/9999999 responds with bad request article doesn't exisit", () => {
+    return request(app)
+      .get('/api/articles/9999999')
+      .expect(400)
+      .then((result) => {
+        expect(result.body.msg).toBe("Bad request, article doesn't exist");
+      });
+  });
```

Comment:
would this test be a 400 or a 404? 
a 400 would be something like /articles/notAnId <- bad, will never work.
a 404 would be something like /articles/82359 <- doesn't exist but may do in the future